bitnami/etcd/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: etcd
-version: 4.4.14
+version: 4.6.0
```

Comment:
Just bump it to 4.5.0

Reply:
My apologies about this, this PR here was actually dependent on PR #1958. Now that #1958 was merged, this change makes more sense. It's a change in the public API that is backwards compatible so I think the bump from 4.5.0 to 4.6.0 makes sense now. Let me know!

Reply:
ok, thanks.
Now that the other PR is merged is not a jump of 2.